src/Analytics/Adapter/ActiveCampaign.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+// Note: ActiveCampaign requires the email prop to be set.
+// It also won't create contacts, it'll only add events to pre-existing contacts.
+
+/**
+ * Utopia PHP Framework
+ *
+ * @package Analytics
+ * @subpackage Tests
+ *
+ * @link https://github.com/utopia-php/framework
+ * @author Torsten Dittmann <torsten@appwrite.io>
+ * @version 1.0 RC1
+ * @license The MIT License (MIT) <http://www.opensource.org/licenses/mit-license.php>
+ */
+
+namespace Utopia\Analytics\Adapter;
+
+use Utopia\Analytics\Adapter;
+use Utopia\Analytics\Event;
+
+class ActiveCampaign extends Adapter
+{
+    /**
+     *  Endpoint for ActiveCampaign
+     *  @var string
+     */
+    public string $endpoint = 'https://trackcmp.net/event';
+
+    /**
+     * Event Key for ActiveCampaign
+     * @var string
+     */
+    private string $key;
+
+    /**
+     * ActiveCampaign actid.
+     * @var string
+     */
+    private string $actid;
+
+    /**
+     * Gets the name of the adapter.
+     * 
+     * @return string
+     */
+    public function getName(): string
+    {
+        return 'ActiveCampaign';
+    }
+
+    /**
+     * @param string $key 
+     * @param string $actid
+     * Adapter configuration
+     * 
+     * @return ActiveCampaign
+     */
+    public function __construct(string $key, string $actid)
+    {
+        $this->key = $key;
+        $this->actid = $actid;
+    }
+
+    /**
+     * Creates an Event on the remote analytics platform.
+     * 
+     * @param Event $event
+     * @return bool
+     */
+    public function createEvent(Event $event): bool 
```

Comment:
@PineappleIOnic This adapter currently does not support creation of contacts ? How are the events added to a contact then ?

Reply:
Active Campaign was not supposed to be added to this PR, my bad. There is a separate PR for active campaign.